reference/sectionB_elements.inc

```diff
@@ -377,44 +377,44 @@ A :code:`math` element information item (referred to in this specification as a
++----------------------------------+----------------------------------------------------------------+
+| Simple Operands                  | :code:`ci`, :code:`cn`, :code:`sep`                            |
++----------------------------------+----------------------------------------------------------------+
+| Basic Structural                 | :code:`apply`, :code:`piecewise`, :code:`piece`,               |
+|                                  | :code:`otherwise`                                              |
++----------------------------------+----------------------------------------------------------------+
+| Relational and Logical Operators | :code:`eq`, :code:`neq`, :code:`gt`, :code:`lt`, :code:`geq`,  |
+|                                  | :code:`leq`, :code:`and`, :code:`or`, :code:`xor`, :code:`not` |
++----------------------------------+----------------------------------------------------------------+
+| Arithmetic Operators             | | :code:`plus`, :code:`minus`, :code:`times`, :code:`divide`,  |
+|                                  | | :code:`power`, :code:`root`, :code:`abs`,                    |
+|                                  | | :code:`exp`, :code:`ln`, :code:`log`,                        |
+|                                  | | :code:`floor`, :code:`ceiling`,                              |
+|                                  | | :code:`min`, :code:`max`, :code:`rem`,                       |
++----------------------------------+----------------------------------------------------------------+
+| Calculus Elements                | :code:`diff`                                                   |
++----------------------------------+----------------------------------------------------------------+
+| Qualifier Elements               | :code:`bvar`, :code:`logbase`, :code:`degree` \*               |
++----------------------------------+----------------------------------------------------------------+
+| Trigonometric Operators          | | :code:`sin`, :code:`cos`, :code:`tan`, :code:`sec`,          |
+|                                  | | :code:`csc`, :code:`cot`,                                    |
+|                                  |                                                                |
+|                                  | | :code:`sinh`, :code:`cosh`, :code:`tanh`, :code:`sech`,      |
+|                                  | | :code:`csch`, :code:`coth`, :code:`arcsin`,                  |
+|                                  | | :code:`arccos`, :code:`arctan`,                              |
+|                                  |                                                                |
+|                                  | | :code:`arcsec`, :code:`arccsc`, :code:`arccot`,              |
+|                                  | | :code:`arcsinh`, :code:`arccosh`, :code:`arctanh`,           |
+|                                  | | :code:`arcsech`, :code:`arccsch`, :code:`arccoth`            |
```

Comment:
Why those two empty lines? Also, I would have either three or six operators per line for each group. I.e.

```
sin, cos, tan, sec, csc, cot,
sinh, cosh, tanh, sech, csch, coth,
arcsin, arccos, arctan, arcsec, arccsc, arccot,
arcsinh, arccosh, arctanh, arcsech, arccsch, arccoth,
```

or

```
sin, cos, tan,
sec, csc, cot,
sinh, cosh, tanh,
sech, csch, coth,
arcsin, arccos, arctan,
arcsec, arccsc, arccot,
arcsinh, arccosh, arctanh,
arcsech, arccsch, arccoth,
```

if the former is too wide.

Reply:
The empty lines were there already ... I presumed they had a purpose of some kind so kept them in.

Reply:
They don't make sense to me, not least because they don't group the operators in any sensible way.